packages/frontend/src/features/metricsCatalog/components/visualization/MetricsVisualization.tsx

```diff
@@ -82,6 +81,17 @@ type Props = {
     isFetching: boolean;
 };
 
+const CHART_COLORS = [
+    'indigo',
+    'red',
+    'green',
+    'blue',
+    'yellow',
+    'orange',
+    'purple',
+    'pink',
+];
+
```

Comment:
this is temporary and my question here is: which colors should we use? ones from the lightdash settings?